.github/workflows/lint-unit.yml

```diff
@@ -77,7 +77,7 @@ jobs:
 
       - name: Prepare database
         run: |
-          bundle exec rails db:migrate
+          bundle exec rails db:reset db:migrate
```

Comment:
`db:reset` is short for `db:drop db:create  db:schema:load  db:seed`. This loads the schema from schema.rb which is not desired here, because we want to test that the migrations are working. 

What you could do if you want to make sure here that the db is re-setup: `rake db:drop db:create  db:migrate`